site/rcuinfo.md

```diff
@@ -1,6 +1,6 @@
 ## Specifying RCU connection information in the model
 
-During creation or updating a domain. There is new section RCUDbInfo in the model under domainInfo section for specifying RCU database connection information without using the command line options -rcu_database and -rcu_prefix.  This is created to support database where the connection string is more complicated and require extra options.
+During creation or updating a domain. There is new section RCUDbInfo in the model under domainInfo section for specifying RCU database connection information without using the command line options -rcu_database and -rcu_prefix.  This is created to support database where the connection string is more complicated and requires extra options.
```

Comment:
Use tic marks to apply `code` font to words and commands, such as `RCUDbInfo`, `domainInfo`, `-rcu_database`, 
 `-rcu_prefix`
**Suggested re-write**:
During creating or updating a domain, there is new section `RCUDbInfo` in the model, under the `domainInfo` section, for specifying RCU database connection information without using the command-line options 
`-rcu_database` and `-rcu_prefix`.  Use this to support a database where the connection string is more complicated and requires extra options.